src/Tag/Base/Tag.php

```diff
@@ -92,8 +92,23 @@ final public function id(?string $id): self
      * @param string|null ...$class One or many CSS classes.
      *
      * @return static
+     *
+     * @deprecated Use {@see addClass()} or {@see replaceClass} instead. Please note that in next major
+     * version method `replaceClass()` will be renamed to `class()`.
      */
     final public function class(?string ...$class): self
```

Comment:
Why not string `btn btn-lg text-primary` string is enough, why do we need an array ?

Reply:
> Why not string `btn btn-lg text-primary` string is enough, why do we need an array ?

Method accept both string and array.